roles/testnode/tasks/apt_systems.yml

```diff
@@ -38,18 +38,24 @@
   args:
     creates: /etc/fstab.bak
   register: add_user_xattr
-  when: modify_fstab == true
+  when:
+    - modify_fstab == true
+    - not containerized_node
 
 - name: Enable xattr for this boot.
   command:
     mount -o remount,user_xattr /
   when: add_user_xattr is defined and
         add_user_xattr is changed
 
-- name: Ensure fuse group exists.
+- name: Ensure fuse, kvm and disk groups exist.
```

Comment:
This tested on baremetal?

Reply:
No but I can do that

Reply:
Tested here https://pulpito.ceph.com/zack-2022-03-11_01:40:50-orch:cephadm:smoke-master-distro-default-smithi/